### ISSUE TITLE
Docs: please provide example of file/image upload (POST)

### ISSUE THREAD

**Issue by mjmare** [plone/plone.restapi#75]
Also, it would be useful to document which fields are available/required when POSTing.

**Comment by lukasgraf** [comment]
I'm working on an update to the docs that explains serialization/deserialzation of non-trivial field types like `NamedFile`.

For now, this is how you do it:

For a field of type `Named[Blob]File` (DX) or `FileField` (AT), serialize the field content as a dictionary with three keys:

- `data` - the base64 encoded contents of the file
- `content-type` - the MIME type of the file
- `filename` - the name of the file, including extension

Example:

```
POST /plone/folder/
Accept: application/json
```

```json

      {
        "@type": "File",
        "title": "My file",
        "file": {
            "data": "TG9yZW0gSXBzdW0uCg==",
            "filename": "lorem.txt",
            "content-type": "text/plain"}
      }
```

**Comment by mjmare** [comment]
Very helpful. Thanks!

**Comment by lukasgraf** [comment]
Closed via #82

**Comment by sarn0ld** [comment]
see #662
